metrics/src/main/java/com/google/gson/metrics/ParseBenchmark.java

```diff
@@ -16,14 +16,6 @@
 
 package com.google.gson.metrics;
 
-import com.google.caliper.Param;
```

Comment:
Google's style rules concerning imports have changed over time, but I think this file follows the current guidelines now and doesn't after these changes. The imports should be in lexicographical order.

Reply:
Sorry, have fixed this now. I was using Eclipse's save actions and had imported the Google Style Guide formatter settings, but had not adjusted the additional Eclipse settings as described in https://github.com/google/styleguide/issues/273#issuecomment-457843194.

Would it make sense to add the [spotless Maven plugin](https://github.com/diffplug/spotless/tree/main/plugin-maven#google-java-format) to enforce the style guide, or was your concern here mainly that the formatting of an existing file was changed?

Reply:
My main concern about doing that would be that we would have to reformat everything to conform, which would invalidate most outstanding PRs. (I have the same concern about updating the tests to JUnit4 + Truth.)

Reply:
Ok, might be best to not apply such a plugin then at the moment.